app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+
+  def new
+  end
+
+  def create
+    admin_user = AdminUser.find_by(email: params[:session][:email].downcase)
+    if admin_user && admin_user.authenticate(params[:session][:password])
+      log_in admin_user
```

Comment:
メソッドに引数を渡すときは `()` で渡すように統一しよう！
```suggestion
      log_in(admin_user)
```

Reply:
@katagiri0306 上記4点ともに修正対応いたしました。
お手数ですが再度確認お願いいたします！